### PR TITLE
fix conflict between replicas with hpa.

### PR DESCRIPTION
**Pull request by morvencao**
Please provide a description for what this PR is for.
resolve: https://github.com/istio/istio/issues/28916

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ X ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[ ] Does not have any changes that may affect Istio users.

### CODE REVIEW

File: releasenotes/notes/28916.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: istioctl
+issue:
+  - 28916
+releaseNotes:
+  - |
+    **Fixed** Fix HPA settings for telemetry is overridden by the inline replicas.
```
Comment (shamsher31) [inline releasenotes/notes/28916.yaml:8]:
```suggestion
    **Fixed** HPA settings for telemetry is overridden by the inline replicas.
```

**Review by shamsher31 — APPROVED** [review]

**Comment by morvencao** [comment]
Looks like `helm` binary is missing in the release-1.7 build-tool:
```
# docker run --rm -it  gcr.io/istio-testing/build-tools:release-1.7-2020-11-13T11-06-08 -- helm version
/usr/local/bin/docker-entrypoint: line 57: exec: helm: not found
```

**Comment by chadlwilson** [comment]
Ahh seems, this does the same as my slightly earlier PR at #28917, although I PRed against`release-1.6` and didn't do the release notes.

@morvencao would it be better for me to cancel my PR and we plan to cherry-pick this into `1.6` instead?

**Comment by morvencao** [comment]
@chadlwilson You can keep your PR since it is against branch `release-1.6`

**Comment by chadlwilson** [comment]
OK, thx @morvencao I'll cherry pick your release notes change. Wasn't aware of how to do that piece 👍

**Review by jwendell — APPROVED** [review]

**Pull request merged**